playbooks/roles/edxapp/defaults/main.yml

```diff
@@ -702,6 +702,17 @@ EDXAPP_SESSION_SAVE_EVERY_REQUEST: false
 
 EDXAPP_SESSION_COOKIE_SECURE: false
 
+EDXAPP_INSTRUCTOR_QUERY_PROBLEM_TYPES:
+  - 'problem',
+  - 'openassessment',
```

Comment:
@stvstnfrd the list of the problem types we want to include came from monica. the specific names come from the way in which the code base distinguishes which xblocks should be queriable, which is a field called "block_type" (https://github.com/Stanford-Online/edx-platform/blob/master/lms/djangoapps/instructor/views/api.py#L1240)